bootstrap/cmd/kfctl/cmd/init.go

```diff
@@ -61,6 +61,7 @@ or a <name>. If just <name>, a directory <name> will be created in the current d
 		}
 
 		useIstio := initCfg.GetBool(string(kftypes.USE_ISTIO))
```

Comment:
wouldn't we want to remove all other cli options in Init like --project, --platform, --namespace, --repo, ...

Reply:
I agree that the plan is to remove all of those flags.
However, as this is PR should not break anything, it will just create a different code path.
Once all platforms migrate, we should be able to delete the old code path.